crates/subspace-networking/src/node.rs

```diff
@@ -234,9 +235,9 @@ impl Node {
     ) -> Result<Pin<Box<dyn Stream<Item = PiecesToPlot>>>, GetPiecesByRangeError> {
         let (result_sender, result_receiver) = oneshot::channel();
 
-        // calculate the middle of the range
-        let f = U256::from_big_endian(&from.0);
-        let t = U256::from_big_endian(&to.0);
+        // calculate the middle of the range (big endian)
+        let f = U256::from(&from.0);
+        let t = U256::from(&to.0);
```

Comment:
Why did `from_big_endian` change to `from`? It would be much better to keep it as `from_big_endian` to avoid ambiguity, we have both big and little-endian use in various places. Hiding `from_big_endian` and `encode` behind `from`/`into` hurts readability a lot and will become a source of bugs and confusion. `from` should not exist on `U256` unless it is generated with `uint` macro already and we can't remove it.

Reply:
`from`is `from_big_endian`. But I will change it back for clarity.